app.py

```
@@ -214,21 +215,33 @@ def kafiya():
-                    error = f"Error processing word: {str(e)}"
+            try:
+                result = kd.lookup(
+                    text_input,
+                    max_per_bucket=_KAFIYA_PER_PAGE_DEFAULT,
+                    page_by_bucket={
+                        "exact": exact_page,
+                        "close": close_page,
+                        "open": open_page,
+                    },
+                ).to_dict()
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)